docs/admin/node.mdx

```diff
@@ -0,0 +1,161 @@
+credentials to `~/.kwil-admin/auth.{cert,key}`.
+
+With this method, the contents of the `auth.cert` file must be appended to the
+node's `clients.pem` file. Specifically,
+
+1. Generate credentials with `./kwil-admin node gen-auth-key`
+2. Securely copy the `~/.kwil-admin/auth.cert` file to the machine where `kwild`
+   is running, if it is not the same host.
+3. Append the contents of the copied `auth.cert` to the node's `clients.pem` file.
+
+  ```sh
+  cat auth.cert >> ~/.kwild/clients.pem
+  ```
+
+If `kwil-admin` is being used on the same machine as `kwild`, you would skip
+copy step and do:
+
+```sh
+cat ~/.kwil-admin/auth.cert >> ~/.kwild/clients.pem
+```
+
+## Commands
+
+The following commands perform node actions via the node's authenticated RPC
+service, configured as described above. All flags are optional if the required
+files are placed in the `~/.kwil-admin` folder.
+
+```
+Usage: kwil-admin node [--rpcserver RPCSERVER] [--authrpc-cert AUTHRPC-CERT]
+  [--tlskey TLSKEY] [--tlscert TLSCERT] <command> [<args>]
+
+Options:
+  --rpcserver RPCSERVER, -s RPCSERVER
+                         admin RPC server address [default: 127.0.0.1:50151]
+  --authrpc-cert AUTHRPC-CERT
+                         kwild's TLS certificate [default: kwild.cert]
+  --tlskey TLSKEY        our TLS key file to establish a mTLS (authenticated)
+                         connection [default: auth.key]
+  --tlscert TLSCERT      our TLS certificate file for server to authenticate us
+                         [default: auth.cert]
+  --help, -h             display this help and exit
+  --version              display version and exit
+```
+
+### Version
+
+The `version` subcommand returns `kwild`'s version string set at compile time.
+For example, a development version of `kwild` may report something similar to
+the following:
+
+```sh
+$ kwil-admin node version
+kwild version 0.5.1-pre+0975d68fb.dirty
+```
+
+### Status
+
+Node status is requested with the `status` subcommand:
+
+For example, `kwil-admin node status` returns JSON formatted status information:
+
+```json
+{
+  "node": {
+    "chain_id": "kwil-chain-nUkVIp",
+    "name": "kwilnode42",
+    "node_id": "9fde182db6dc5c2d5a08f395cdc9320a76a4f15d",
+    "proto_ver": 8,
+    "app_ver": 0,
+    "block_ver": 0,
+    "listen_addr": "tcp://0.0.0.0:26656",
+    "rpc_addr": "tcp://0.0.0.0:26657"
+  },
+  "sync": {
+    "app_hash": "51ffe4001668c0ea7791a2e8d647fe88d7542f2594e91382ca0a7026db6a1571",
+    "best_block_hash": "9f7d2fc800e3926e04db7ae72778faa47cb551394662dcc131824203efa9bbd8",
+    "best_block_height": 27462,
+    "best_block_time": "2023-09-19T15:46:36.693-05:00",
+    "syncing": false
+  },
+  "current_validator": {
+    "pubkey": "d659e53ff403c52a1ae85445c51dd714d57e75b4ae6fa2e009a9082bfef870db",
+    "pubkey_type": "ed25519",
+    "power": 1
+  }
+}
+```
+
+### Peers
+
+A list of the peers currently connected to the node are reported with the
+`peers` subcommand.
+
+... TODO EXAMPLE output
```

Comment:
I actually haven't run the peers subcommand on a testnet with >1 node.  Right now I just get back `[]` for my single node network.  :)